src/cli.cr

```diff
@@ -67,14 +67,33 @@ module Shards
         when "version"
           Commands::Version.run(args[1]? || path)
         else
-          display_help_and_exit(opts)
+          program_name = "shards-#{args[0]}"
+          if Process.find_executable(program_name)
+            run_shards_subcommand(program_name, args)
+          else
+            display_help_and_exit(opts)
+          end
         end
 
         exit
       end
     end
   end
 
+  def self.run_shards_subcommand(process_name, args)
+    Process.exec(
+      command: process_name,
+      args: args[1..-1],
+      env: nil,
+      clear_env: false,
+      shell: false,
+      input: STDIN,
+      output: STDOUT,
+      error: STDERR,
+      chdir: nil
```

Comment:
These are redundant

```suggestion
```

Reply:
Given that shards is an important tool, I opted for explicitly using these arguments rather that falling back to the default. This would avoid a problem if the defaults were ever changed. 

That being said, I'm happy to make this change if you think it's a positive change! 😄

Reply:
Given that wound be a breaking change I don't see a need for being explicit that much - in the end using the defaults has some merits too (like using `:redirect` pipe instead of explicitly passing `STD*`)